darts/models/exponential_smoothing.py

```diff
@@ -9,15 +9,16 @@
 from .forecasting_model import UnivariateForecastingModel
 from ..logging import get_logger
 from ..timeseries import TimeSeries
+from .. import ModelMode
 
 logger = get_logger(__name__)
 
 
 class ExponentialSmoothing(UnivariateForecastingModel):
     def __init__(self,
-                 trend: Optional[str] = 'additive',
+                 trend: Optional[ModelMode] = ModelMode.ADDITIVE,
                  damped: Optional[bool] = False,
-                 seasonal: Optional[str] = 'additive',
+                 seasonal: Optional[ModelMode] = ModelMode.ADDITIVE,
```

Comment:
Are the underlying string values of the enum still supported as argument values?

Reply:
No, the raw strings are not suported anymore. We only accept the Enum members.
But to be backward-compatible, it can be interesting to accept both.